client-v2/src/main/java/com/clickhouse/client/api/internal/HttpAPIClientHelper.java

```diff
@@ -50,25 +55,48 @@ public class HttpAPIClientHelper {
 
     private RequestConfig baseRequestConfig;
 
+    private String proxyAuthHeaderValue;
+
     public HttpAPIClientHelper(Map<String, String> configuration) {
         this.chConfiguration = configuration;
-        this.httpClient = createHttpClient(configuration, null);
+        this.httpClient = createHttpClient();
         this.baseRequestConfig = RequestConfig.custom()
                 .setConnectionRequestTimeout(1000, TimeUnit.MILLISECONDS)

```

Comment:
Should it be configurable?

Reply:
Yes. Added more configuration parameters. Planning to add more later.